### DISCUSSION TITLE
post/bytedance-interview-failed-2023-09

### DISCUSSION THREAD

**Discussion by giscus** [nickxudotme/nickxudotme.github.io#5] [General]
# post/bytedance-interview-failed-2023-09

字节一面  自我介绍 简单介绍字节青训营项目 是组队的吗 项目耗时 项目收获的点 ELK 是你们搭建的吗 ELK 的软件安装 数据流大概是怎样的 是通过什么写到 Logstash 里的 Logstash 的功能 你们用的的 fail2ban 是什么 traceID 介绍 微服务框架用的什么 trac

https://nickxu.me/post/bytedance-interview-failed-2023-09

**Comment by fanenr** [comment]
加油！

**Reply by jarvis657** [reply]
加油，看了你的文章，作为一个编码10多年的老鸟，感觉还是不错的。继续加油持之以恒！

**Reply by nickxudotme** [reply]
谢谢，我今天刚腾讯 HR 面完，要等一个星期审批，大概是稳了（
最近会补一些没写完的博客 ❤️
